template/en/default/bug/new_bug.html.tmpl

```diff
@@ -5,16 +5,29 @@
 # This Source Code Form is "Incompatible With Secondary Licenses", as
 # defined by the Mozilla Public License, v. 2.0.%]
 
-[% PROCESS "global/field-descs.none.tmpl" %]
-
+[% PROCESS global/variables.none.tmpl %]
 [% title = BLOCK %]Enter [% terms.Bug %] [% END %]
-
-[% PROCESS global/header.html.tmpl
-  title = title
-  generate_api_token = 1
-  no_yui = 1
-%]
-
+[% IF !header_done %]
+  [% PROCESS bug_modal/common_header.html.tmpl 
+    title = title
+  %]
+  [% PROCESS global/header.html.tmpl %]
+  [% header_done = 1 %]
+[% END %]
```

Comment:
the header_done stuff is from post_bug/show_bug because they do... weird things with the markup. Remove the header_done conditions -- this template is only meant to be called on its own